### PR TITLE
feature: introduce server keepAliveTimeout into config files

### PR DESCRIPTION
**Pull request by rostislav-simonik**
**Type:**  feature
**Description:**
Set default timeout to 60 seconds (it's workaround for issue reported in #301).

### CODE REVIEW

File: src/lib/bootstrap.js [hunk 1]
```diff
@@ -89,7 +89,10 @@ function startVerdaccio(config: any,
       } else { // http
         webServer = http.createServer(app);
       }
-
+      if (config.server && config.server.keepAliveTimeout) {
+        // $FlowFixMe library definition for node is not up to date (doesn't contain recent 8.0 changes)
+        webServer.keepAliveTimeout = config.server.keepAliveTimeout;
```
[the remark this answers is not in the document]
Comment (rostislav-simonik) [reply src/lib/bootstrap.js]: Node 6,7 won't be affected, it adds keepAliveTime attribute into instance, but won't be used at all. It's little bit hacky.  Even somebody backports given attribute into 6,7, which probably never happens, I expect that will work the same way as in node 8.

[the remark this answers is not in the document]
Comment (juanpicado) [reply src/lib/bootstrap.js]: Cool, thanks for the detailed PR, I just wanna be sure 👍 Thanks so much. I like the new conf setup. 

I'd leave an example comment on #301 in order people run into this issue know show to update their config file accordingly.

**Comment by juanpicado** [comment]
Pushed, CI will publish the new version in a while, coffee time.

File: src/lib/bootstrap.js (hunk 1 above)
Comment (fhp) [inline src/lib/bootstrap.js]: This PR sets the timeout in seconds, while the [node docs](https://nodejs.org/api/http.html#http_server_keepalivetimeout) suggest that the value is in microseconds.

```suggestion
        webServer.keepAliveTimeout = config.server.keepAliveTimeout * 1000;
```

Comment (juanpicado) [reply src/lib/bootstrap.js]: Good catch, could you PR?

Comment (fhp) [reply src/lib/bootstrap.js]: yes, on it

Comment (fhp) [reply src/lib/bootstrap.js]: #1221

**Pull request merged**